libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -346,6 +359,49 @@ public QnAMakerDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
         [JsonProperty("rankerType")]
         public StringExpression RankerType { get; set; } = new StringExpression(RankerTypes.DefaultRankerType);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to include precise answer in response. 
+        /// </summary>
+        /// <value>
+        /// True or false to include precise answer in response, defaults to false. 
+        /// </value>
+        [JsonProperty("enablePreciseAnswer")]
+        public bool EnablePreciseAnswer { get; set; }
```

Comment:
Use BoolExpression here instead of native type to enable future integration with composer if needed